### PR TITLE
Add interactive remote mode

### PR DESCRIPTION
**Pull request by triwav**

### CODE REVIEW

File: src/cli.ts [hunk 1]
```diff
@@ -148,6 +149,14 @@ void yargs
         return new SendTextCommand().run(args);
     })
 
+    .command('interactive', 'Provides a way to send a series of ECP key events similar to how Roku Remote Tool works but from the command line', (builder) => {
```
Comment (TwitchBronBron) [inline src/cli.ts:152]: Chris and I played around with some alternative names. How about:

`remote-control` 

So usage would be:

```
npx roku-deploy remote-control
```

We could add an alias like `RC` or `R` or something if that feels verbose?

Comment (triwav) [reply src/cli.ts]: updated. Added `rc` shorter version as well

File: src/commands/RemoteControl.ts [hunk 2]
```diff
@@ -0,0 +1,104 @@
+import * as readline from 'readline';
+import type { RokuKey } from '../index';
+import { rokuDeploy, util } from '../index';
+
+export class RemoteControl {
```
Comment (TwitchBronBron) [inline src/commands/RemoteControl.ts:5]: Let's call this class `RemoteControlCommand` and the file `RemoteControlCommand.ts` to align with the other files in this folder.

**Review by TwitchBronBron — APPROVED** [review]
Once you rename the class, then this can be merged. Nice work!

**Pull request merged**